### PR TITLE
Removes the Transformation Sting ability

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Removes the Transformation Sting ability, and the `NOTRANSSTING` trait.

Pretty much the same thing as in #14589, but I figured that since that's been open for six months now with no change, and that the only way to reverse the sting was removed (#15821), it'd be best to do this separately.

## Why It's Good For The Game

To be honest, the reason for removing this should be pretty obvious to anyone who's played more than two Changeling rounds.

While someone could in theory use the transformation sting to frame someone else, I've never seen that happen once in 1100 hours.
There's even advanced rules specifically because of it being abused so often:
> Murderboning so you can absorb is seen as excessive. Your abilities should be used to further your goals or to prevent apprehension if you are found out. Example: Transformation Sting.
Randomly transforming crew for no other purpose than to cause chaos runs counter to this, however using it to create copies of yourself to throw off pursuit of Security is acceptable.

And even despite that, it still happens to some extent almost every cling round.

## Changelog
:cl:
del: Removed the Transformation Sting ability.
/:cl:

### CODE REVIEW

**Comment by Mitchs98** [comment]
> and that the only way to reverse the sting was removed (#15821

I'd like to point out this is actually untrue. Genetic syringes can reverse it all the same for races that SM could reverse it for.

That said. I'm all for it being removed. All I ever see it really used for is grief like SM, very rarely do I see it actually used for its purpose.

**Comment by AffectedArc07** [comment]
> and that the only way to reverse the sting was removed (#15821)

image
image

You cannot argue this as a removal reason, when you yourself are the removal reason. Thats like me nerfing the shit out of skrell then removing them because "Its just a human clone now".

**Comment by matttheficus** [comment]
GriefSting is not fun to deal with, nor have I seen it be used for its actual purpose in nearly 1000 hours. Id be ecstatic to see it go.

**Comment by SabreML** [comment]
> You cannot argue this as a removal reason, when you yourself are the removal reason. Thats like me nerfing the shit out of skrell then removing them because "Its just a human clone now".

Yeah I'm not trying to use that as an excuse to remove the transformation sting; I was just as against the ability before that PR was merged as I am now. Even dismissing that point I still believe there's more than enough reasons to get rid of this.

That was meant as more of a side note and another point against the transformation sting, similar to the Shadowling mindshield nerf. That change made people dislike Shadowlings even more than they already did, but it's only one of many reasons that there's a PR to remove the gamemode entirely.
The only reason I didn't make this PR before or as part the SM one is because it was already a feature in #14589, and I didn't want to just steal from that.

**Review by Kyep — APPROVED** [review]
Yes, kill it.

**Review by variableundefined — COMMENTED** [review]
Two maintainers objection. Closing.

**Pull request closed without merging**
